### ISSUE TITLE
Can't pull from library

### ISSUE THREAD

**Issue by abstahl** [matrix-msu/kora-connect#7]
Ann Stahl here...I've managed to pull a couple of files (1 image and 1 pdf) into a library and tried to create a page on which those files can be accessed. All seems to go well; I add them to the page and the following shortcode is inserted in the editor: 

[KORAGALLERY KG_TYPE= 'infscroll' KG_IMAGECONTROL= 'Image File' KG_AUDIOCONTROL= 'default' KG_VIDEOCONTROL= 'default' KG_TITLECONTROL= 'Family' KG_DESCCONTROL= 'Date of Photo' KGIS_PAGESIZE='5' SID = '885' QUERY= '(kid,=,86-375-0),or,(kid,=,86-374-1)'][/KORAGALLERY]

Trouble is neither show up on the page :(   Instead when I view page I get "No results from RestfulAPI";

Tried again asking for only one file and managed to get them up, though now with very awkward appearance and spacing with code as follows:

[KORAGALLERY KG_TYPE= 'infscroll' KG_IMAGECONTROL= 'Image' KG_AUDIOCONTROL= 'default' KG_VIDEOCONTROL= 'default' KG_TITLECONTROL= 'Family name' KG_DESCCONTROL= 'Date collected' KGIS_PAGESIZE='5' SID = '884' QUERY= 'kid,=,86-374-1'][/KORAGALLERY][KORAGALLERY KG_TYPE= 'infscroll' KG_IMAGECONTROL= 'Image File' KG_AUDIOCONTROL= 'default' KG_VIDEOCONTROL= 'default' KG_TITLECONTROL= 'Family' KG_DESCCONTROL= 'Location' KGIS_PAGESIZE='5' SID = '885' QUERY= 'kid,=,86-375-0'][/KORAGALLERY]

Help!

**Comment by seilagonzalez** [comment]
well working on it, but you have two calls for one gallery. You should remove shortcode before you want to include a new one.

**Comment by seilagonzalez** [comment]
[KORAGALLERY KG_TYPE= 'infscroll' KG_IMAGECONTROL= 'Image File' KG_AUDIOCONTROL= 'default' KG_VIDEOCONTROL= 'default' KG_TITLECONTROL= 'Family' KG_DESCCONTROL= 'Location' KGIS_PAGESIZE='5' SID = '885' QUERY= 'kid,=,86-375-0'][/KORAGALLERY]

This one worked!

**Comment by seilagonzalez** [comment]
And this one too:
[KORAGALLERY KG_TYPE= 'infscroll' KG_IMAGECONTROL= 'Image' KG_AUDIOCONTROL= 'default' KG_VIDEOCONTROL= 'default' KG_TITLECONTROL= 'Family name' KG_DESCCONTROL= 'Date collected' KGIS_PAGESIZE='5' SID = '884' QUERY= 'kid,=,86-374-1']

**Comment by watrall** [comment]
@seilagonzalez identified a couple of things that are causing this issue.  I'm going to let her and @catherinefoley describe the big one.  The small one is that you _have_ to have a paragraph break between two shortcodes (you've got two shortcodes stacked up right next to one another)

**Comment by catherinefoley** [comment]
This is the short code for a gallery with several records. We will use this for the structure for displaying multiple records on Ann's site. [KORAGALLERY KG_TYPE= 'infscroll' KG_IMAGECONTROL= 'default' KG_AUDIOCONTROL= 'default' KG_VIDEOCONTROL= 'fileuploadmp4' KG_TITLECONTROL= 'titleDisplay' KG_DESCCONTROL= 'descriptionContent' KGIS_PAGESIZE='5' SID = '245' QUERY= '(kid,eq,2D-F5-1A4),or,(kid,eq,2D-F5-184),or,(kid,eq,2D-F5-14),or,(kid,eq,2D-F5-1),or,(kid,eq,2D-F5-10),or,(kid,eq,2D-F5-104),or,(kid,eq,2D-F5-105),or,(kid,eq,2D-F5-107),or,(kid,eq,2D-F5-10D)'] [/KORAGALLERY]

**Comment by catherinefoley** [comment]
[KORAGALLERY KG_TYPE= 'infscroll' KG_IMAGECONTROL= 'Image File' KG_AUDIOCONTROL= 'default' KG_VIDEOCONTROL= 'default' KG_TITLECONTROL= 'Subject' KG_DESCCONTROL= 'Date of  Photo' KGIS_PAGESIZE='5' SID = '885' QUERY= '(kid,eq,86-375-6),or,(kid,eq,86-375-5),or,(kid,eq,86-375-10),or,(kid,eq,86-375-11),or,(kid,eq,86-375-13),or,(kid,eq,86-375-12)'] [/KORAGALLERY]

**Comment by seilagonzalez** [comment]
do they work?

**Comment by seilagonzalez** [comment]
I am will create ticket with odd stuff that I see.

**Comment by abstahl** [comment]
Here is what I am using currently to pull up single images

[KORAGALLERY KG_TYPE= 'infscroll' KG_IMAGECONTROL= 'Image File' KG_AUDIOCONTROL= 'default' KG_VIDEOCONTROL= 'default' KG_TITLECONTROL= 'Family' KG_DESCCONTROL= 'Location' KGIS_PAGESIZE='1' SID = '885' QUERY= 'kid,=,86-375-1'][/KORAGALLERY]

**Comment by catherinefoley** [comment]
Here is the short code with all of the images from Ann's Kora scheme called "Images". There are 19 records.

[KORAGALLERY KG_TYPE= 'pagination' KG_IMAGECONTROL= 'Image File' KG_AUDIOCONTROL= 'default' KG_VIDEOCONTROL= 'default' KG_TITLECONTROL= 'Family' KG_DESCCONTROL= 'Date of Photo' KGIS_PAGESIZE='4' SID = '885' QUERY= '(kid,=,86-375-5),or,(kid,=,86-375-14),or,(kid,=,86-375-12),or,(kid,=,86-375-13),or,(kid,=,86-375-6),or,(kid,=,86-375-0),or,(kid,=,86-375-1),or,(kid,=,86-375-10),or,(kid,=,86-375-11),or,(kid,=,86-375-4),or,(kid,=,86-375-F),or,(kid,=,86-375-E),or,(kid,=,86-375-D),or,(kid,=,86-375-C),or,(kid,=,86-375-B),or,(kid,=,86-375-9),or,(kid,=,86-375-A),or,(kid,=,86-375-8),or,(kid,=,86-375-7)'][/KORAGALLERY]

**Comment by abstahl** [comment]
OK...I've done a test page using Catherine's short code which pulled all images up to a single page. However, when I try to edit it to include the image for just one family (86-375-A), I get an error on a new page that I created for that family, using the following: 

[KORAGALLERY KG_TYPE= 'pagination' KG_IMAGECONTROL= 'Image File' KG_AUDIOCONTROL= 'default' KG_VIDEOCONTROL= 'default' KG_TITLECONTROL= 'Family' KG_DESCCONTROL= 'Date of Photo' KGIS_PAGESIZE='4' SID = '885' QUERY='(kid,=,86-375-A)'][/KORAGALLERY]

I also tried copying and pasting code that works for the Chorigboono House page onto a new page--it doesn't work when pasted--either for the original picture 86-375-1 nor when I changed the '1' to 'A' for Gbangmbe House's picture. 

Very frustrating!
